contracts/test/stealth-key-registry-test.ts

```diff
@@ -0,0 +1,185 @@
+import hre, { ethers } from 'hardhat';
+const { isAddress } = ethers.utils;
+import { Artifact } from 'hardhat/types';
+
+import { StealthKeyRegistry } from '../typechain';
+
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import chai, { expect } from 'chai';
+
+const { deployContract, solidity } = hre.waffle;
+chai.use(solidity);
```

Comment:
Huh, what does this do? Guessing it adds Waffle chai matchers? Just curious since we've used their matchers without needing this in the past, I think from `import '@nomiclabs/hardhat-waffle'` in the hardhat config